Html часть

Serega020 opened the pull request:
криво косо ну как то так

Serega020 committed:
html

Домашка

Serega020 committed:
Изменение теста

тест

Serega020 committed:
Изменения для теста

Serega020 committed:
кодировка

Dina0308 reviewed (commented):
очень даже неплохо!) добавлены дата атрибуты в нужных местах для автопроверки. комментрии:
1. нужно обернуть лого в хедере в ссылку
2. лишний тег header в header - почему-то с картинкой. нужно уточнить что это и зачем:)
3. есть теги <P> - лучше прописывать строчную букву.
4. каждый пункт преимущество оформлен в отдельный список - лучше сделать один список для всех преимуществ. (каждое преимущество будет li)
5. нужно внимательнее посмотреть список жилье-еда-сувениры. напутано с тегами. можно сделать так: ul, в котором три li (для каждого из пунктов Еда-жилье-сувениры), в каждом из которых есть теги p с названием и субтекстом
6. вижу мысль добавить подписку в футер - это хорошо (т.к. подписка, как и футер повторяется на каждой странице), но ее нужно очень сильно доработать)

Serega020 committed:
Исправление

1.Логотип в хедере обрернул в ссылку
2.убрал лишний тег хедер
3.поменял тег р на строчную
4.преимущества сделал в один список

Serega020 committed:
Изменил футер

Serega020 committed:
Обернул логотип Html Academy  в ссылку

Serega020 committed:
Изменения небольшие

Serega020 committed:
Вложил картинки в список

Serega020 committed:
Редактирование

Serega020 committed:
Плохо, но как то так

Serega020 committed:
Update Catalog.html

Serega020 committed:
Добавил скрытый заголовок

Serega020 committed:
Delete Catalog.html

Serega020 committed:
Create catalog.html

Serega020 committed:
Убрал таблицу и сделал остальные доработки

Serega020 committed:
Merge pull request #1 from htmlacademy-htmlcss/master

Merge pull request #1 from Serega020/master

Serega020 committed:
Разметка и графика

1.скачал изображения(сжал их)
2.подключил контентные изображения
3.отредактировал разметку

Serega020 committed:
Merge branch 'master' of https://github.com/Serega020/2425687-sedona-38

Dina0308 reviewed (commented):
Намного лучше, комментарии:
1. сразу бросается в глаза русское название файлов - не должно быть + заглавные буквы в названиях файлов (тоже не должно быть)
2. в каталоге - форма включает не только чекбоксы и радиокнопки, но и ввод цены - нужно вклюичть в эту же форму (и мин макс заменить на от и до)
3. в каталоге кнопки в секции фильтров объединить в блок - для дальнейшего позиционирования на странице.
4. с карточками гостиниц все ок, кроме того, что внимательнее нужно посмотреть теги (нет закрывающего тега li)
5. "Подписаться" в секции рассылки должно быть надписью кнопки, не отдельный спан (т.е. включить эту подпись в тег кнопки_
6. картинки выгружены из макета, но не подключены в верстке, можно уже добавить пути)
7. попробуйте использовать prettier - код станет упорядоченнее и самому будет удобнее смотреть, если где-то пропущен тег

Serega020 committed:
исправления

Постарался максимально все сделать
